fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```diff
@@ -99,6 +104,15 @@ def before_save(self):
             self.update_published_status()
         self.set_route()
 
+    def create_email_groups(self):
+        for group in [
+            "Event Participants",
+            "CFP Proposers",
+            "Accepted Proposers",
+            "Rejected Proposers",
```

Comment:
The list of options here differs from the earlier list of email groups. If this is intentional, I'm not sure why it is so.

Reply:
We should not be needing to create Chapter Master and Other group at this moment. The first should happen at chapter create and Other option should be used in future when introducing custom group function